### PR TITLE
Fix descriptions starting with formula name.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/gnu-go.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class GnuGo < Formula
-  desc "GNU Go"
+  desc "Free program that plays the game of Go"
```
Comment (claui) [inline Formula/gnu-go.rb:2]:
```suggestion
  desc "Plays the game of Go"
```

File: Formula/makeself.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class Makeself < Formula
-  desc "Make self-extractable archives on UNIX"
+  desc "Script that generates a self-extractable compressed tar archive"
```
Comment (claui) [inline Formula/makeself.rb:2]:
```suggestion
  desc "Generates a self-extractable compressed tar archive"
```

File: Formula/maxwell.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 class Maxwell < Formula
-  desc "Maxwell's daemon, a mysql-to-json kafka producer"
+  desc "Application that reads MySQL binlogs and writes row updates as JSON to Kafka"
```
Comment (claui) [inline Formula/maxwell.rb:2]:
```suggestion
  desc "Reads MySQL binlogs and writes row updates as JSON to Kafka"
```

**Review by claui — COMMENTED** [review]
Other than those few minor suggestions, LGTM.

File: Formula/makeself.rb (hunk 2 above)
Comment (gromgit) [reply Formula/makeself.rb:2]: I think "self-extracting" is the commonly-used adjective for such archives.

Comment (claui) [reply Formula/makeself.rb:2]: @gromgit Good catch!

**Review by claui — DISMISSED** [review]

**Review by SMillerDev — DISMISSED** [review]

**Review by bayandin — CHANGES_REQUESTED** [review]
@reitermarkus could you please rearrange the changes into single-formula-per-commit

**Comment by bayandin** [comment]
> ⚠️ @SMillerDev bottle publish [failed](https://github.com/Homebrew/homebrew-core/actions/runs/231453862).

This is a `CI-syntax-only` PR, so the bottles weren't even produced: [`Error: No artifact with the name `bottles` was found!`](https://github.com/Homebrew/homebrew-core/runs/1048148071?check_suite_focus=true#step:5:37). So it should be ok to merge it via GitHub UI.

**Comment by reitermarkus** [comment]
> could you please rearrange the changes into single-formula-per-commit

@bayandin, do you have a script for doing that? Otherwise I don't think it is worth enforcing this for bulk changes like this.

**Comment by dtrodrigues** [comment]
https://github.com/MikeMcQuaid/dotfiles/blob/master/bin/git-commit-each

**Review by SMillerDev — APPROVED** [review]

**Pull request merged**